nuxeo-features/nuxeo-platform-notification/nuxeo-platform-notification-core/src/main/java/org/nuxeo/ecm/platform/ec/notification/email/EmailAuthenticator.java

```diff
@@ -32,32 +32,20 @@ public EmailAuthenticator(Properties properties) {
 
     @Override
     protected PasswordAuthentication getPasswordAuthentication() {
-        String user = value("user");
+        String user = protocolValue("user");
         if (user == null) {
             return null;
         }
-        String password = value("password");
+        String password = protocolValue("password");
         if (password == null) {
             return null;
         }
         return new PasswordAuthentication(user, password);
     }
 
-    protected String value(String name) {
-        String value = protocolValue(name);
-        if (value != null) {
-            return value;
-        }
-        return defaultValue(name);
-    }
-
     protected String protocolValue(String name) {
         String key = "mail." + getRequestingProtocol() + "." + name;
         return properties.getProperty(key);
     }
 
-    protected String defaultValue(String name) {
```

Comment:
are you sure it's unused? it seems designed for being implemented by a subclass

Reply:
Yes, according to a grep in the platform and released addons repositories.